### PR TITLE
add .net sample code for aws covid-19 datalake

### PR DESCRIPTION
**Pull request by ulili5**
*Issue #, if available:*

*Description of changes:*
Updated documentation to add Tutorial of .NET Sample code that uses COVID-19 Data Lake
https://registry.opendata.aws/aws-covid19-lake/ 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 486f77896dbbffc08985762e665f1e1200b31634
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=59bde049-7d9a-458d-97aa-c99d1a3d2923%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: ca10dcc75d5db68b6eb4682056a44b913d8c6eaf
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=2d613661-f4ab-4615-ac89-58ff7202234f%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 728a891e00f9607d458bdb8856fc553b38b0d63b
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=45bbfd86-9736-48cd-be41-8273c6d6b733%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: c6d6b6690005524e779b4e8ac7a3d3d9e80be7ed
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=ccfa88a8-ec3b-481a-8c17-b845eae6c7de%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: services.yaml [hunk 1]
```diff
@@ -12,3 +12,5 @@
 - QuickSight
 - EC2
 - S3
+- AWS SDK for .NET
```
Comment (jflasher) [inline services.yaml:15]: Please drop the SDK from this list as I think it's not a useful service callout.

File: services.yaml [hunk 2]
```diff
@@ -12,3 +12,5 @@
...
+- AWS Cloud​Formation
```
Comment (jflasher) [inline services.yaml:16]: Please shorten to just CloudFormation to match other services.

**Review by jflasher — CHANGES_REQUESTED** [review]
Thanks @ulili5! Left two comments, but then should be good to merge.

File: services.yaml (hunk 2 above)
Comment (ulili5) [reply services.yaml]: @jflasher done!!!

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 5109f3d5f70c76bfeaa9d7b6505a0d0860c874c2
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=9cff3300-05f1-4ea3-8384-2412315445a5%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 3efdb3d00171a6143db9a8c385b9470e8e04e05d
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=dd41c6e7-5e0a-4c31-8d37-32c9c6afb37b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by jflasher — APPROVED** [review]

**Pull request merged**